docs/theory/apportion_abundance.md

```diff
@@ -0,0 +1,65 @@
+(apportion-abundance)=
+# Apportioning kriged abundance
+
+{{<b>Perhaps a link to the [biomass apportionment page](../theory/apportion_biomass.md) or additional (brief) references/re-explanations here?</b>}}
+
+Summed biomass estimates along $\ell$ and $\alpha$ for each $s$ along transect interval $k$ for aged ($B_{s, \ell, \alpha}^{k,\textrm{aged}}$) and unaged ($B_{s, \ell}^{k,\textrm{unaged}}$) kriged biomass estimates are summed together ($\hat{B}_{s}^{k}$) by first summing $B_{s, \ell, \alpha}^{k,\textrm{aged}}$ across $\alpha$ for each $\ell$ and then adding that quantity with $B_{s, \ell}^{k,\textrm{unaged}}$. The result is then summed across all $\ell$ to yield $\hat{B}_{s}^{k}$. This is done in tandem with the unsexed aged ($B_{\ell, \alpha}^{k,\textrm{aged}}$) and unaged ($B_{\ell}^{k,\textrm{unaged}}$) kriged biomass estimates to yield the total biomass ($\hat{B}^{k}$). This not calculated via adding male ($\hat{B}_{M}^{k}$) and female ($\hat{B}_{F}^{k}$) since $\hat{B}^{k}$ is inclusive of male, female, <i>and</i> unsexed fish.
+
+![ image ](../images/initial_biomass_summation.jpg)
+
+These biomass estimates are then converted to sexed ($\hat{N}_{s}^k$) and total ($\hat{N}^k$) abundance by using averaged length-weight regression output ($\overline{W}^i(\ell)$) for stratum $i$. Consequently, $\overline{W}^i(\ell)$ can be defined either by using the average length-weight relationship produced from each $i$ or parameterizing $\overline{W}^i(\ell)$ with the mean length ($\bar{\ell}^i$) within each $i$. It is important to note, however, that both $\hat{N}_{s}^k$ and $\hat{N}^k$ are calculated using $\overline{W}^i(\ell)$ fit from <b>all</b> individuals (i.e. male, female, and unsexed).
```

Comment:
For the last sentence:
> It is important to note, however, that both $\hat{N}_{s}^k$ and $\hat{N}^k$ are calculated using $\overline{W}^i(\ell)$ fit from <b>all</b> individuals (i.e. male, female, and unsexed).

I wonder if this is an unintended error...?

Reply:
It certainly is possible that this is an unintended bug in the code. Perhaps using the general relationship from all fish/samples may be more statistically 'robust' for strata where few individuals from a particular sex (aged or unaged) may be available. I can also take a second look through this section of the code within the loop to see if there are any more context clues (e.g. comments that say 1 thing, but the code says a different thing).